### ISSUE TITLE
KB navigation Master-detail changes

### ISSUE THREAD

**Issue by zdrawku** [IgniteUI/igniteui-angular#7048]
## Description  
With the new keyboard navigation enhancements certain changes should be made to the Master-detail:
- [ ] [Specification ](https://github.com/IgniteUI/igniteui-angular/wiki/Grid-Master-Detail)- This will no longer apply

> 6. When click on a Masterdetail and press Ctrl+arrow down/up the grid is not scrolled

- [x] Tests - The previous state behavior was that on Ctrl + up/down, the grid will be scrolled up and down. Now with the new changes, this won't happen.
